package.json

```diff
@@ -53,7 +53,7 @@
     "@octokit/graphql": "^4.8.0",
     "@octokit/plugin-paginate-rest": "^2.17.0",
     "@octokit/rest": "^18.12.0",
-    "@playwright/test": "^1.33.0",
+    "@playwright/test": "^1.49.1",
```

Comment:
It's not clear to me why it was related, but attempting to run tests against Vite 6 raised some errors that are fixed in later versions of Playwright.